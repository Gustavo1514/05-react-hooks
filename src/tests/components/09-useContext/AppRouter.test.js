const { shallow, mount } = require('enzyme')
const {
  default: AppRouter
} = require('../../../components/09-useContext/AppRouter')
const { UserContext } = require('../../../components/09-useContext/UserContext')

describe('Pruebas sobre <AppRouter />', () => {
  const user = {
    id: 1,
    name: 'Henry'
  }
  const wrapper = mount(
    <UserContext.Provider
      value={{
        user
      }}
    >
      <AppRouter />
    </UserContext.Provider>
  )
  test('debe mostarse correctamente ', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
