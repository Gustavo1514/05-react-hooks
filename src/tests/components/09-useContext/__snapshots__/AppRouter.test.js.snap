// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre <AppRouter /> debe mostarse correctamente  1`] = `
<div>
  <ul
    className="nav"
  >
    <li
      className="nav-item"
    >
      <a
        className="nav-link active"
        href="/"
        onClick={[Function]}
      >
        Use Context
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link active"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link "
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link "
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
    </li>
  </ul>
  <div
    className="container"
  >
    <div>
      <h1>
        HomeScreen
      </h1>
      <hr />
      <pre>
        {
   "id": 1,
   "name": "Henry"
}
      </pre>
    </div>
  </div>
</div>
`;
